examples/README.md

```diff
@@ -121,11 +121,25 @@ steps:
     FROM topFantasyMovies
     WHERE title = ${myFavoriteMovie}
 output:
-- dataFrameName: moviesWithRatings
+- dataFrameName: topFantasyMovies
   outputType: Parquet
   outputOptions:
     saveMode: Overwrite
-    path: moviesWithRatings.parquet
+    path: topFantasyMovies.parquet
+- dataFrameName: topFantasyMovies
+  outputType: CSV
+  outputOptions:
+    saveMode: Overwrite
+    path: topFantasyMovies.csv
+    coalesce: true
+    extraOptions:
+      multiline: "true"
+- dataFrameName: topFantasyMovies
+  outputType: JSON
+  outputOptions:
+    saveMode: Overwrite
+    path: topFantasyMovies.jsonl
+    repartition: 10
```

Comment:
Hey, can you remove the two new data outputs (csv and json)? I think that in the readme it's OK to leave only one example